### ISSUE TITLE
double underline renders wrongly

### ISSUE THREAD

**Issue by jakov** [tanakahisateru/js-markdown-extra#12]
```
__strong__, __double__snake, is__double__snake, endsnake__double__
```

renders as

```
<strong>strong</strong>, __double__snake, is__double__snake, endsnake<strong>double</strong>
```

although in php markdown you get the following:

```
<strong>strong</strong>, <strong>double__snake, is__double__snake, endsnake__double</strong>
```

**Comment by tanakahisateru** [comment]
This issue will be fixed by `MarkdownExtra_Parser` in 1.2 branch soon.

**Comment by tanakahisateru** [comment]
Branch 1.2 was merged to master. Currently it seems fixed.
